src/plugins/stake-plugins/types.ts

```diff
@@ -191,10 +192,10 @@ export const filterStakePolicies = (policies: StakePolicy[], filter?: StakePolic
   if (filter == null) return policies
 
   let out: StakePolicy[] = [...policies]
-  const { currencyCode, wallet } = filter
+  const { currencyCode, pluginId } = filter
 
-  if (wallet != null) {
-    out = out.filter(policy => [...policy.rewardAssets, ...policy.stakeAssets].some(asset => asset.pluginId === wallet.currencyInfo.pluginId))
+  if (pluginId != null) {
```

Comment:
`wallet` is still part of this `StakePolicyFilterType`. Should still support the
original behavior here by extracting the pluginId from the wallet, or else it's
confusing that there's a noop prop on the filter when you call `filterStakePolicies`